src/Makefile

```diff
@@ -57,7 +57,7 @@ GCCOPT   += -O3 -ffast-math -fPIC -fno-exceptions -fno-rtti
 ICCOPT   += -O3 -fast -fPIC -fno-exceptions
 CLANGOPT += -O3 -ffast-math -fPIC
 else ifeq ($(COMPILER_MODE),DEBUG)
-GCCOPT   += -O3 -g -Wall -Wno-unknown-pragmas -Wno-reorder -fbounds-check -fPIC  #-fprofile-arcs -ftest-coverage
+GCCOPT   += -O3 -g -Wall -Wno-unknown-pragmas -Wno-reorder -fbounds-check -fPIC  -fno-rtti #-fprofile-arcs -ftest-coverage
```

Comment:
This was because I was trying to use dynamic_cast when testing and I hadn't noticed that the reason that it was compiling was because the -no-rtti option was missing. Thus the reasib for  adding it to the debug code is to make sure that same code compiles with both options